src/api/status/src/views/builds.hbs

```diff
@@ -92,4 +92,4 @@
 <script src="https://cdn.jsdelivr.net/npm/xterm-addon-fit@0.5.0/lib/xterm-addon-fit.min.js"></script>
 <script src="https://cdn.jsdelivr.net/npm/xterm-addon-web-links@0.4.0/lib/xterm-addon-web-links.min.js"></script>
 <script src="https://cdn.jsdelivr.net/npm/xterm-addon-webgl@0.11.3/lib/xterm-addon-webgl.min.js"></script>
```

Comment:
All this other JS should get done in the bundle step vs. included as separate files.  You can do that in a follow-up if you like, but you need to file it now if so.

Reply:
Yeah I'll do that in a follow-up. There'll be a `main.js`, and it also utilizes Vite for sass so we won't need a separate scss compiler anymore.

Reply:
Great, file an issue so we don't lose track.